crates/nu-command/tests/commands/redirection.rs

```diff
@@ -288,3 +288,38 @@ fn redirection_should_have_a_target() {
         );
     }
 }
+
+#[test]
+#[cfg(not(windows))]
```

Comment:
Would be great if we could convert this test to something that reliably runs on all platforms.

Reply:
I'm thinking a way to make such output in our testbin.
I'd prefer to make it in separate pr, cause it requires a lot of changes in tests